azurerm/resource_arm_redis_cache_test.go

```diff
@@ -640,6 +640,28 @@ func TestAccAzureRMRedisCache_SubscribeAllEvents(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMRedisCacheWithoutAuth(t *testing.T) {
```

Comment:
in order to match the other test cases can we make this:

```suggestion
func TestAccAzureRMRedisCache_withoutAuth(t *testing.T) {
```